api/server/handlers/release/update_rollback.go

```
@@ -118,7 +118,9 @@ func updateReleaseRepo(config *config.Config, release *models.Release, helmRelea
 		return fmt.Errorf("Could not find field repository in config")
 	}
 
-	if repoStr != release.ImageRepoURI {
+	if repoStr != "public.ecr.aws/o1j4x7p4/hello-porter" &&
```

Should throw an error to the user here